rpcs3/Emu/system_progress.cpp

```diff
@@ -141,7 +141,7 @@ void progress_dialog_server::operator()()
 					break;
 				}
 
-				if (show_overlay_message)
+				if (show_overlay_message && g_cfg.misc.show_ppu_compilation_hint)
```

Comment:
This is wrong.
It has to be checked in show_ppu_compile_notification(), otherwise you'll just show a real progress dialog instead, which is even more annoying than the overlay

Reply:
Thanks, fixed